app/src/main/java/com/jerboa/db/AppDBMigrations.kt

```diff
@@ -384,6 +384,25 @@ val MIGRATION_26_25 =
         }
     }
 
+val MIGRATION_26_27 =
+    object : Migration(26, 27) {
+        override fun migrate(db: SupportSQLiteDatabase) {
+            db.execSQL(UPDATE_APP_CHANGELOG_UNVIEWED)
+            db.execSQL(
+                "ALTER TABLE AppSettings ADD COLUMN swipe_to_action_preset INTEGER NOT NULL DEFAULT 1",
+            )
+        }
+    }
+
+val MIGRATION_27_26 =
+    object : Migration(27, 26) {
+        override fun migrate(db: SupportSQLiteDatabase) {
+            db.execSQL(
+                "ALTER TABLE AppSettings DROP COLUMN swipe_to_action_preset",
```

Comment:
```
android.database.sqlite.SQLiteException: near "DROP": syntax error (code 1 SQLITE_ERROR): , while compiling: ALTER TABLE AppSettings DROP COLUMN swipe_to_action_preset
                                                                                                    	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
                                                                                                    	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1045)
```

It seems that DROP operations aren't supported

@dessalines should we remove all these downgrade migrations? They don't work anyway

Reply:
Ooh. It's a surprise to me that SQLite doesn't support drop column in tables. It can be done via creating a separate table without that column (i looked into it and i guess that's how others do it)

I think separate PR should be created for downgrade migrations fix

Reply:
yeah probably, but the down migrations aren't really used. So it won't be priority

Reply:
Yeah its totally fine to remove them. One of the worst android bugs in existence: https://stackoverflow.com/questions/8045249/how-do-i-delete-column-from-sqlite-table-in-android